docs/RFCS/README.md

```diff
@@ -1,209 +1,94 @@
-   implementer) of the RFC. For instance, this can happen when an
-   expert is not available during the initial review (e.g. because she
-   is on vacation).
-
-9. Once the changes described in the RFC have been made, change the
-   status of the PR from `in-progress` to `completed`. If subsequent
-   developments render an RFC obsolete, set its status to `obsolete`.
-
-   When you mark a RFC as obsolete, ensure that its text references the
-   other RFCs or PRs that make it obsolete.
-
-# Overhead of the RFC process
-
-There is a trade-off between the impetus to make
-more time for decision-making and consensus-reaching, mandating
-minimum wait times to let every participant evaluate the work, and
-the impetus to move fast and often discover/fix problems later.
-
-At the time of this writing (2017), this RFC process is biased towards
-moving faster, correspondingly spelling out a minumum delay of only
-one week in the RFC process with a 1-2 working day FCP.
-
-# RFC Status
-
-During its lifetime an RFC can have the following status:
-
-- Draft
-
-  A newly minted RFC has this status, until either the proposal is
-  accepted (next possible status: in-progress), deferred (next possible
-  status: postponed) or that it is DOA (next possible status: rejected).
-
-- Rejected
-
-  A RFC receives this status when the PR discussions have concluded
-  the proposal should not be implemented.
-
-  Next possible status: none. If you want to ressucitate an idea, make
-  a new RFC.
-
-- In-progress
-
-  A RFC receives this status when the PR discussions have concluded
-  the proposal should be implemented, and the RFC is ready to commit
-  (merge) to the main repository.
-
-  Next possible status: completed (when the work is done), rejected
-  (proposal not implemented/implementable after all), obsolete (some
-  subsequent work removes the need for the feature).
-
-- Completed
-
-  A RFC receives this status when the described feature has been
-  implemented. It may stay with this status indefinitely or until made
-  obsolete.
-
-- Obsolete
-
-  A RFC receives this status when the described feature has been
-  superseded.
-
-- Postponed
-
-  A RFC receives this status when the PR discussions have concluded that
-  due to implementation complexity, lack of customer demand, or other
-  issues with the proposal, the work should be postponed to a later date.
-
-  Next possible status: draft (the decision has been made to reconsider
-  this proposal), rejected (proposal not implemented/implementable after all),
-  obsolete (some subsequent work removes the need for the feature).
+6. Once the changes in the RFC have been implemented and merged,
+   change the `Status` field of the document from `in-progress` to
+   `completed`. If subsequent developments render an RFC obsolete,
+   change its status to `obsolete`. When you mark a RFC as obsolete,
+   ensure that its text references the other RFCs or PRs that make it
+   obsolete.
```

Comment:
I think it's worth having a "postponed" status to reflect projects that were postponed due to opportunity cost.